clang/lib/StaticAnalyzer/Checkers/ChrootChecker.cpp

```diff
@@ -104,15 +146,35 @@ void ChrootChecker::evalChdir(const CallEvent &Call, CheckerContext &C) const {
     R = R->StripCasts();
     if (const StringRegion* StrRegion= dyn_cast<StringRegion>(R)) {
       const StringLiteral* Str = StrRegion->getStringLiteral();
-      if (Str->getString() == "/")
-        state = Mgr.addGDM(state, ChrootChecker::getTag(),
-                           (void*) JAIL_ENTERED);
+      if (Str->getString() == "/") {
```

Comment:
I think we could just inline `Str`.
```suggestion
      if (StrRegion->getStringLiteral()->getString() == "/") {
```

Reply:
Understood, will address.